pandas/core/frame.py

```diff
@@ -4704,6 +4704,7 @@ def sort_values(
         inplace=False,
         kind="quicksort",
         na_position="last",
+        ignore_index=False,
```

Comment:
can you type as bool_t as well

Reply:
just a question, I am wondering if it is possible to add type annotation here? as long as I add annotation, the `type validation` in CI will fail, the error message is: 
```
pandas/core/series.py:2691: error: Signature of "sort_values" incompatible with supertype "NDFrame"
pandas/core/frame.py:4701: error: Signature of "sort_values" incompatible with supertype "NDFrame"
```
@jreback @WillAyd

Reply:
I think `sort_values` currently has some structural issues that prevent any part of it being annotated. Ran into the same issue here:

https://github.com/pandas-dev/pandas/pull/27237/files/9ce3b26a0cccf8fe6042a9e60e8cec982a3b9565#diff-1e79abbbdd150d4771b91ea60a4e1cc7

@charlesdong1991 can you open an issue for this as a follow up?

Reply:
sure, then in this PR, shall I skip annotation for now? and in follow-up PR, once the problem is addressed and fixed, then annotation can be added

Reply:
Yea I think that is reasonable

Reply:
not sure if it is caused by 
```python
@Substitution(**_shared_doc_kwargs)
@Appender(NDFrame.sort_values.__doc__)
```
but as long as i see this above functions in `frame.py`, there is no annotation

Reply:
FYI, I opened issue #30451 as a follow-up @WillAyd